### DISCUSSION TITLE
let copy/cut cell(s) in command mode doesn't affect clipboard

### DISCUSSION THREAD

**Discussion by yuuuxt** [microsoft/vscode-jupyter#11473] [New Ideas]
In vscode when copying/cutting cell(s) in command mode, it affects the clipboard (putting the cell contents in clipboard).

I hope it behaves  (or provides an option to behave) the clipboard doesn't change, which is the default behavior of jupyter notebook. 

A sample use cases is to duplicate a cell and change some strings/variables of the duplicated cell. 
- In jupyter notebook, user can (find and) copy the wanted string first, then select the target cell in command mode and press C and V to duplicate, and click to select and paste;
- Currently in vscode, user can only duplicate cell first, then goes to find and copy the wanted string, then goes back to replace.

(Will this affect copying and pasting cells across notebooks,  #1480?)

**Comment by IanMatthewHuff** [comment]
Interesting, I don't believe that I've looked before at how Jupyter handled the cutting / pasting of cells (regarding the clipboard). If you don't mind, I'm going to transition this discussion to a new feature request issue. Seems reasonable for us to consider this as a work item (discussions are more for taking about usage of the product, this seems like a possible fix, and I'm worried we'll lose track of it here).

**Reply by yuuuxt** [reply]
sure :)
